Assets/LoggingManager/LogStore.cs

```diff
@@ -0,0 +1,276 @@
+﻿using System;
+using System.CodeDom;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using Boo.Lang.Runtime;
+using UnityEngine;
+
+
+public enum LogType
+{
+    Normal,
+    Meta
+}
+
+public class LogStore : MonoBehaviour
+{
+
+    private SortedDictionary<string, List<string>> logs;
```

Comment:
This is a dictionary that sorts the keys in alphabetic order. 
At first, i wanted to find a dictionary that keeps the order of insertion of the keys but i didn't find one.

The reason is use this datatype is because when we log the string on the fly, we have to make sure that each log is logged in the right order before appending them to the current big string.

Let's imagine that the logs are the following : 
`Millis;framecount;Email\n`

Now, we need to add our logs to the string in the exact order of the headers :
`2547;10;test@test.dk\n`

Thus, if the logs are not added in the right order, they will be reorganized with this dictionary because each logs is added to this dictionary before being converted to a string.